lib/PriceSettings.js

```diff
@@ -33,54 +37,80 @@ class PriceSettings extends Component {
+    } = this.props;
     const formattedChain = chain === 'bitcoincash' ? 'Bitcoin Cash' : chain;
 
     return (
-      <div className={'col'}>
-        <Header type="h3">Select your price settings</Header>
-        <Text type="p">
-          These settings will update the price display across your app. The
-          cryptocurrency is based on the node you are currently connected to (
-          <span style={{ textTransform: 'capitalize' }}>
-            {formattedChain || '...'}
-          </span>
-          ).
-        </Text>
-        <Text>
-          {price.crypto +
-            ' ' +
-            price.fiatSymbol +
-            this.moneyNumber(price.price) +
-            ' '}
-        </Text>
-        <select
-          value={price.fiat}
-          onChange={e => {
-            updateFiat(e.target.value);
-            refreshPrice();
-          }}
-        >
-          {price.availableFiats.map(val => (
-            <option value={val} key={val}>
-              {val}
-            </option>
-          ))}
-        </select>
-        &nbsp;
-        <select
-          value={price.feed}
-          onChange={e => {
-            updateFeed(e.target.value);
-            refreshPrice();
-          }}
-        >
-          {price.availableFeeds.map(val => (
-            <option value={val} key={val}>
-              {val}
-            </option>
-          ))}
-        </select>
+      <div className="col">
+        <div className="row mb-3">
+          <Header type="h3">Select your price settings</Header>
+          <Text type="p">
+            These settings will update the price display across your app. The
+            cryptocurrency is based on the node you are currently connected to (
+            <span style={{ textTransform: 'capitalize' }}>
+              {formattedChain || '...'}
+            </span>
+            ).
+          </Text>
+          <Text>
+            {price.crypto +
+              ' ' +
+              price.fiatSymbol +
+              this.moneyNumber(price.price) +
+              ' '}
+          </Text>
+          <select
+            value={price.fiat}
+            onChange={e => {
+              updateFiat(e.target.value);
+              refreshPrice();
+            }}
+          >
+            {price.availableFiats.map(val => (
+              <option value={val} key={val}>
+                {val}
+              </option>
+            ))}
+          </select>
+          &nbsp;
+          <select
+            value={price.feed}
+            onChange={e => {
+              updateFeed(e.target.value);
+              refreshPrice();
+            }}
+          >
+            {price.availableFeeds.map(val => (
+              <option value={val} key={val}>
+                {val}
+              </option>
+            ))}
+          </select>
+        </div>
+        <div className="row">
+          <Header type="h5">Settings for Widget Display</Header>
+          <Label
+            text="Show settings menu?"
```

Comment:
Kinda think you can lose this question -- there's triple redundancy explaining that button right there!

<img width="449" alt="screen shot 2019-01-14 at 11 42 49 pm" src="https://user-images.githubusercontent.com/2084648/51165759-25168e00-1856-11e9-8252-55ecd1436889.png">